### ISSUE TITLE
[Bug] Separation force is wrong

### ISSUE THREAD

**Issue by stergiosba** [stergiosba/PaxRL#5]
Tested on toy example with MATLAB and the naive python and get different outcomes:

Naive Python (same output as MATLAB):

def separate(aid, distance_mat, positions, velocities, neighbors):
    desired_separation = 30.0
    neighbors_aid = neighbors[aid]
    steer = jnp.array([0,0])

    for i,bool_neig in enumerate(neighbors_aid):
        if aid!=i:
            difference = positions[aid] - positions[i]  #vector from the agent to other agent in the neighbourhood
            difference = difference/la.norm(difference)   #normalizing to get unit vector
            difference = difference/distance_mat[aid,i]
            steer = steer + difference #i>1)+length(boids)*difference*(i==1);

    count = jnp.sum(neighbors_aid)
    if count > 0:
        steer = steer/count

    if la.norm(steer) > 0:
        steer = steer/la.norm(steer)
        steer = steer - velocities[aid]
        steer = steer/la.norm(steer)

    return steer

Test scenario parameters to reproduce if needed:

    n=5
    key = random.PRNGKey(1)
    key2 = random.PRNGKey(2)
    A = random.uniform(key,shape = (n,2))
    V = random.uniform(key2,shape = (n,2))

**Comment by stergiosba** [comment]
Fixed with new version
